composer.json

```diff
@@ -34,6 +34,9 @@
         "behatch/contexts": "^2.6",
         "doctrine/doctrine-bundle": "^1.6.3",
         "doctrine/orm": "^2.5.2",
+        "doctrine/mongodb-odm": "^1.1.5",
+        "doctrine/mongodb-odm-bundle": "~3.0",
+        "ext-mongo": "*",
```

Comment:
Weird, try to enable both but it should require only `ext-mongodb` for php7+ (see linked travis docs).